packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -540,13 +538,19 @@ Future<int> buildAndroid(
   }
 
   int result = _buildApk(platform, buildMode, components, flxPath, keystore, outputFile);
+  status.stop(showElapsedTime: true);
+
   if (result == 0) {
+    double size = new File(outputFile).lengthSync() / (1024 * 1024);
```

Comment:
Last time this came up we mused that dart:io should have a FileSize
abstraction similar to Duration. :) Nothing came of said musings to my
knowledge.

Thanks for fixing.
On Saturday, May 28, 2016, Devon Carew <notifications@github.com> wrote:

> In packages/flutter_tools/lib/src/commands/build_apk.dart
> <https://github.com/flutter/flutter/pull/4263#discussion_r64995533>:
>
> >    if (result == 0) {
> > +    double size = new File(outputFile).lengthSync() / (1024 * 1024);
>
> I made a utility method in base/utils.dart.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/flutter/flutter/pull/4263/files/7c96fafe91f243e43102aa672488fe5caad859af#r64995533>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALTvi0nReERxXch0rs-DeVfqohUC5U7eks5qGMswgaJpZM4IpI_t>
> .
>